usage/cli/src/main/license/source-inclusions.yaml

```diff
@@ -20,5 +20,5 @@
 # extras file for org.heneveld.license-audit-maven-plugin
 # listing projects from which *source* files are included
 
-- id: underscore.js
+- id: underscore.js:1.7.0
```

Comment:
Just a query on the plugin - I can see this ties up with the entry in the overrides.yaml, its not clear how/where this file gets picked up from the [license-audit-maven-plugin](https://github.com/ahgittin/license-audit-maven-plugin) readme?